README.md

```diff
@@ -54,6 +60,12 @@ git fetch origin
 git checkout main
 </pre>
 
+#### Point WP CLI to the right place
+
+Check that [WP CLI](https://wp-cli.org/) is able to connect to your local WordPress installation. From the command line prompt in your local WordPress directory (mentioned above), run `wp plugin update`. If it works, it will show a table of installed plugins.
```

Comment:
Same here:

```suggestion
Check that [WP CLI](https://wp-cli.org/) is able to connect to your local WordPress installation. From the command line prompt in your local WordPress directory (mentioned above), run: 
```sh
wp plugin update
```
 If it works, it will show a table of installed plugins.
```